smalltalksrc/VMMaker/StackInterpreter.class.st

```diff
@@ -4404,32 +4404,30 @@ StackInterpreter >> codeGeneratorToComputeAccessorDepth: aCCodeGenerator [
 ]
 
 { #category : #'indexing primitive support' }
-StackInterpreter >> commonAt: stringy [
+StackInterpreter >> commonAt [
```

Comment:
I suggest not changing this method. We know that works and manage all the cases.

Just create the specialised versions and this one will be dead eventually.